sql/updates/world/master/RoG_2017_09_22_00_world.sql

```diff
@@ -0,0 +1,5 @@
+-- Fix for: 	Revenge - (6572, 5302, 5301)
+-- Class:		Warrior
+-- Spec:		Protection
+DELETE FROM `spell_proc` WHERE `SpellId` = 5301;
```

Comment:
Recommended TC format for this SQL:
```sql
-- Protection Warrior
DELETE FROM `spell_proc` WHERE `SpellId` = 5301; -- Revenge (6572, 5302, 5301)
INSERT INTO `spell_proc` (`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`) VALUES
(5301, 0, 0, 0, 0, 0, 0, 0, 0, 0, 48, 0, 0, 0, 0, 0);
```
- remove apostrophes from the numbers (integers)
- place the `VALUES` row on its separate line (readability)
- keep the top comment line short and to the point
- specific details can be put in comment behind the query